docs/user/otp-deployment.md

```diff
@@ -0,0 +1,14 @@
+# OTP Deployment Guide
+
+## OTP Deployment Architecture
+
+The figure below depicts the OTP deployment architecture.
+
+![OTP Deployment Diagram](../img/otp-deployment-diagram.png)
```

Comment:
A few comments on this diagram:

- The nesting of the UI components doesn't quite make sense except for the files being in the s3 bucket. I'd change each of the items to be at the same level except for the files in the s3 bucket.
- Cloudfront is optional, so I would annotate it with an asterisk
- The nesting doesn't quite make sense for the OTP backend as well because it's technically possible to have an ELB without Route 53.
- There is only one Java jar file in the deployment, but I'd also add that there is a deployment bundle that consists of GTFS feeds and OSM data
- ELB deployments are still an optional configuration so I'd note that with an asterisk
- What is "SRC"?
- There shouldn't be an arrow from "bounding box" to "OTP UI Mod Files" as the OTP UI is configured independently of datatools-server

I think it would be good to have a paragraph or two explaining the main components of this deployment process and what they are. It'd be great if each labeled box in the diagram had a corresponding text description to clarify its meaning.